src/lib/bootloader/grub2base.rb

```diff
@@ -120,6 +120,9 @@ def propose
 
       propose_serial
 
+      # we proposing new config and we do not know sections yet, so propose empty
+      @sections.default = ""
```

Comment:
Isn't this a bug in the constructor of Sections? IMHO this code should go to https://github.com/yast/yast-bootloader/blob/f519e4818cecaea9f25c52e63a41b7b0692be0b5/src/lib/bootloader/sections.rb#L20